### PR TITLE
MILESTONE 1: Initialize bookstore

### PR DESCRIPTION
**Pull request by Rachelwebdev**
### In this milestone, I;

- Created the todo app using create react app
- Set up linters for react/redux
- Added a descriptive README file
- Implemented the correct use of gitflow
- Used Best practices in my code
- Set up React Router and added two routes to the app: `books` and `categories`
- Created a `Navbar` component that includes `Link`s to the two routes
- Created a `Book` component that displays a single book
- Created a `Form` component with two inputs
- Rendered all the components and added basic styling to it
- Fixed linter errors

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <>
+      <li className={styles.singleBook}>
+        <p>title</p>
+        <p>author</p>
+        <button type="button">remove</button>
+      </li>
+    </>
+  );
+}
```
Comment (DammyShittu) [inline src/components/Book.js]: - [ ] The Book component should take in `title` and `author` as props and render them.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (Rachelwebdev) [reply src/components/Book.js]: I would include that now. Thanks for taking note of it.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Rachelwebdev 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rachelwebdev** [comment]
Thanks for the approval

**Pull request merged**